terraform/development/main.tf

```diff
@@ -29,7 +29,7 @@ data "aws_subnet_ids" "development" {
   vpc_id = data.aws_vpc.development_vpc.id
   filter {
     name   = "tag:Type"
-    values = ["private"]
+    values = ["public"]
   }
 }
```

Comment:
Sorry, I confused things by giving too many options.
Because you made the change below on line 42 harcoding the subnet-id the change to "public" isn't needed anymore.

In fact, you could delete lines 28-34 because you're no longer using that resource to find the subnets.

Reply:
I have removed la resource